src/custom/constants/lists.ts

```diff
@@ -94,7 +94,7 @@ export const DEFAULT_ACTIVE_LIST_URLS_BY_NETWORK: NetworkLists = {
   [ChainId.RINKEBY]: [COW_DAO_LIST, RINKEBY_LIST],
   // [ChainId.ROPSTEN]: [GEMINI_LIST],
   [ChainId.XDAI]: [COW_DAO_LIST, HONEY_SWAP_XDAI],
-  // [ChainId.GOERLI]: [GEMINI_LIST],
+  [ChainId.GOERLI]: [GEMINI_LIST], // TODO: should we add here goerli list as for rinkeby
```

Comment:
Yes, we should have one just like we do for rinkeby.
Not sure who created that but should be another task to do this

Reply:
Added a PR for this https://github.com/cowprotocol/cowswap/pull/569, the token addresses are for RInkeby for now.